fs/vfs/fs_ioctl.c

```diff
@@ -103,6 +103,13 @@ int file_vioctl(FAR struct file *filep, int req, va_list ap)
         ret = file_fcntl(filep, F_SETFD,
                          file_fcntl(filep, F_GETFD) & ~FD_CLOEXEC);
         break;
+
+      case FIOC_FILEPATH:
+        if (!INODE_IS_MOUNTPT(inode))
+          {
+            ret = inode_getpath(inode, (FAR char *)arg);
```

Comment:
```suggestion
            FAR char *path = (FAR char *)(uintptr_t)arg;
            ret = inode_getpath(inode, path);
```
CI is failing on a warning:
>Error: vfs/fs_ioctl.c:110:40: error: cast to pointer from integer of different size [-Werror=int-to-pointer-cast]

Reply:
Done.